Makefile

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env make
```

Comment:
Perhaps comment at the top saying what the purpose of this makefile is?  i.e. to build the docker image.

If you could leave the Makefile where it was and rung it with `make -f` to avoid polluting the top level?

(You can also remove the #! line from the top, I don't think anyone expects to run the Makefile like that).

Reply:
All done with commit https://github.com/emscripten-core/emsdk/pull/571/commits/531f6c8996ac8e1411bc77b6516845847acb5cfa.